Adds the Motorhead random NPC class, audits scientist and doctor

I-am-Erk opened the pull request:
<!-- HOW TO USE: Under each "#### Heading" below, enter information relevant to your pull request.
Leave the headings unless they don't apply to your PR.

Please read carefully and don't delete the comments delimited by "< !--" and "-- >"
Once a pull request is submitted, automatic stylistic and consistency checks will be performed on the PR's changes.
The results of these can be either seen under the "Files changed" section of a PR or in the check's details.

NOTE: Please grant permission for repository maintainers to edit your PR.  It is EXTREMELY common for PRs to be held up due to trivial changes being requested and the author being unavailable to make them. -->

#### Summary
Content "Adds Motorhead NPC class and adjusts scientists and doctors"
<!-- This section should consist of exactly one line, edit the one above.
1. Replace the word "Category" with one of these words: Features, Content, Interface, Mods, Balance, Bugfixes, Performance, Infrastructure, Build, I18N.
2. Replace the text inside the quotes with a brief description of your changes.
Or if you don't want a changelog entry, replace the whole line with just the word "None" (with no quotes).
For more on the meaning of each category, see:
https://github.com/CleverRaven/Cataclysm-DDA/blob/master/doc/CHANGELOG_GUIDELINES.md
If approved and merged, your summary will be added to the project changelog:
https://github.com/CleverRaven/Cataclysm-DDA/blob/master/data/changelog.txt -->

#### Purpose of change
We could very much use more dynamic NPC classes with better, more reasonable starting skills and proficiencies. @John-Candlebury suggested that I add a race car enthusiast, who can possibly direct you to some unique things.

#### Describe the solution
This starts the process by creating a random NPC class that will eventually be able to give you that quest. If I get another break, I'll start adding the dialogue in this PR, but if not this can stand on its own.

In the process of reviewing skills and proficiencies to balance the class, I noticed some big problems in the skillset for a few others, most notably doctor and scientist. the errors were too egregious to ignore - Scientists weren't getting Chemistry! - so I audited them and, in the process, added proficiencies for the classes as well.

#### Describe alternatives you've considered
I considered differentiating "driver" and "mechanic" motorheads, but I think that's getting too specific for now. 
Later on, we might have a mechanic, just like we should probably develop different types of scientists.

#### Testing
I'm on  holiday and out of time, testing will have to wait.

#### Additional context
Vroom vroom.

I-am-Erk committed:
Create NC_MOTORHEAD.json

item groups

I-am-Erk committed:
Update NC_MOTORHEAD.json

forgot these were hard coded

I-am-Erk committed:
Update classes.json

Add motorhead and fix up scientist and doctor

I-am-Erk committed:
Update classes.json

Add proficiencies to doctor, scientist, and motorhead

I-am-Erk committed:
Update NC_MOTORHEAD.json

Add a weapon selection

I-am-Erk committed:
generify starting stories... for now.

Paprika-H commented:
Why was the cowboy class removed?

Paprika-H commented on data/json/npcs/NC_MOTORHEAD.json:
```diff
@@ -0,0 +1,99 @@
+    "subtype": "distribution",
+    "entries": [
+      { "item": "jeans", "prob": 70 },
+      { "item": "pants", "prob": 10 },
```
```suggestion
      { "item": "pants", "prob": 10 },
      { "item": "pants_leather", "prob": 25 },
      { "item": "motorbike_pants", "prob": 15 },
```

Paprika-H commented on data/json/npcs/NC_MOTORHEAD.json:
```diff
@@ -0,0 +1,99 @@
+      { "item": "hoodie", "prob": 15 },
+      { "item": "jacket_light", "prob": 15 },
+      { "item": "jacket_windbreaker", "prob": 15 },
+      { "item": "jacket_leather", "prob": 20 },
```
```suggestion
      { "item": "jacket_leather", "prob": 20 },
      { "item": "jacket_leather_mod", "prob": 5 },
```

Paprika-H commented on data/json/npcs/NC_MOTORHEAD.json:
```diff
@@ -0,0 +1,99 @@
+    "type": "item_group",
+    "id": "NC_MOTORHEAD_shoes",
+    "subtype": "distribution",
+    "entries": [ { "item": "sneakers", "prob": 60 }, { "item": "boots", "prob": 40 } ]
```
```suggestion
    "entries": [ { "item": "sneakers", "prob": 20 }, { "item": "boots", "prob": 60 }, { "item": "motorbike_boots", "prob": 30 }  ]
```

Holli-Git reviewed (commented):
Adds leather pants, motorbike pants, motorbike boots, armored leather jacket to possible clothes to wear. Also adjusted shoes so boots were more common than sneakers

I-am-Erk commented:
> Why was the cowboy class removed?

Misclick of some kind I suspect. Did this on my phone outside by the pool, surprised there aren't more. Should be able to fix it in a couple minutes.

Paprika-H commented:
> Misclick of some kind I suspect. Did this on my phone outside by the pool, surprised there aren't more. Should be able to fix it in a couple minutes.

Subconscious cowboy bigotry

I-am-Erk commented:
Damn, my biases are laid naked

Paprika-H commented:
Also, "Motorhead" is slang for meth addict, would it make sense for these people to be on drugs, and interested in selling/buying alongside vehicle?

ampersand55 commented:
New game goal: recruit a motorhead named Lemmy and give him a handlebar mustache.

I-am-Erk committed:
lint1

I-am-Erk committed:
lint2 and undelete

I-am-Erk committed:
a few more clothes suggested by paprika_h

I-am-Erk committed:
another clothing suggestion from paprika_h

I-am-Erk committed:
Merge branch 'master' into motorhead-npc

I-am-Erk committed:
move THUG back to common classes.json

While it's used by hell's raiders, it's a dynamic NPC class and should be kept with the others.

I-am-Erk commented:
This is ready to test when I get home, or if someone wants to try in the meantime, cool.
